### PR TITLE
 Remove empty line after client.CmdInspect docstring

### LINKED ISSUE #12706: Remove empty line after client.CmdInspect docstring
subj

### PR DESCRIPTION
**Pull request by jmzwcn**
fix #12706 
Signed-off-by: Daniel Zhang jmzwcn@gmail.com

### CODE REVIEW

File: api/client/inspect.go [hunk 1]
```diff
@@ -13,7 +13,6 @@ import (
 )
 
 // CmdInspect displays low-level information on one or more containers or images.
-//
 // Usage: docker inspect [OPTIONS] CONTAINER|IMAGE [CONTAINER|IMAGE...]
```
Comment (runcom) [inline api/client/inspect.go]: I think the line which should be removed is this one

**Comment by icecrime** [comment]
@jmzwcn Yes, @runcom is right it's line 18 that needs to be removed ;-) Can you please update the PR?

File: api/client/inspect.go [hunk 2]
```diff
@@ -13,9 +13,7 @@ import (
 )
 
 // CmdInspect displays low-level information on one or more containers or images.
-//
```
Comment (runcom) [inline api/client/inspect.go]: As the other client *.go files are, this line should be left to have the same format

**Comment by jmzwcn** [comment]
Have followed @runcom advice, commit again.

**Comment by runcom** [comment]
@jmzwcn np! I think this is good now

**Comment by LK4D4** [comment]
@jmzwcn You need squash your commits to one :)

**Comment by jfrazelle** [comment]
let us know if you need help

**Comment by jmzwcn** [comment]
Thanks! I will merge three commits into one soon.

**Comment by thaJeztah** [comment]
Thanks! LGTM

**Comment by duglin** [comment]
LGTM

**Pull request merged**